JS/Lesson14/onder/script.js

```diff
@@ -9,71 +9,99 @@ const randomNumberFrom0To10 = Math.floor(Math.random()*10+1);
+// I don't know which one is best approach but I just tried my logic :/
+while( stringExercise10.includes("can help")){
+    stringExercise10 = stringExercise10.replace("can help", "help");
+    console.log("BONUS Exercise",stringExercise10);
+}
 
 console.log('------------------------');
 // 11. Use the `startsWith` method to check if strings below starts with a price, and print the result to the console.
 const string1Exercise11 = "$9.99 - Sale! 50% off all items in store";
 const string2Exercise11 = "Bundle deal: Save $50 when you buy two products together";
+console.log("Exercise11: ", string1Exercise11.startsWith("$"));
+console.log("Exercise11: ", string2Exercise11.startsWith("$"));
 
 console.log('------------------------');
 // 12. Use the `endsWith` method to check if a string ends with a "!" (exclamation mark), and print the result to the console.
 const string1Exercise12 = "$9.99 - Sale! 50% off all items in store";
 const string2Exercise12 = "Bundle deal: Save $50 when you buy two products together!";
-
+console.log("Exercise 12: ", string1Exercise12.endsWith("!"));
+console.log("Exercise 12: ", string2Exercise12.endsWith("!"));
 
 console.log('------------------------');
 // 13. Use the `includes` method to check if a string includes a "%" character and print the result to the console.
 const string1Exercise13 = "$9.99 - Sale! 50% off all items in store";
 const string2Exercise13 = "Bundle deal: Save $50 when you buy two products together!";
-
+console.log("Exercise13: ", string1Exercise13.includes("%"));
+console.log("Exercise13: ", string2Exercise13.includes("%"));
 
 console.log('------------------------');
 // 14. Use the `substring` method to extract the word "Databases" from string stringExercise14 based on two indices and print it to the console.
 // Question: What is the difference between `substring()` and `slice()`?
 const stringExercise14 = 'Databases are an essential component of many applications.';
+console.log("Exercise14: ", stringExercise14.substring(0,8));
+// Answer: substring method's second parameter represents how many letters of word wanted to be extracted. (Not the whole sentence)
+// Substringin ikinci parametresi çıkartılmak istenen kelime ile ilgilidir. Slice methodunda cümlenin kaçıncı harfleri olacağı seçilirken
+// substring metodunda ise başlangıç harfi cümleden seçildikten sonra ikinci parametre o seçilen harften sonra kaç harf seçileceği ile ilgilidir.
 
 console.log('------------------------');
 // 15. Remove extra spaces from the string stringExercise15 and print the result to the console.
 const stringExercise15 = "    Hello world    ";
+console.log("Exercise15: ", stringExercise15.trim());
 
 console.log('------------------------');
 // 16. Extract the price and currency from a string below using "slice" method and print it to console.
 const planPriceString = "Premium plan - 9.99 USD/month."; // Should return "9.99 USD"
+console.log("Exercise16: ", planPriceString.slice(15,23));
```

Comment:
This works as long as you know the exact position of characters however in real life you usually don't know it.
In real life, I'd use a regular expression to solve it but we don't know it yet. For now, I'd recommend to use a solution similar to Tuğba's :D 
`const extractedPriceCurrency = planPriceString.slice(planPriceString.indexOf("- "), planPriceString.indexOf("/"));`